engine/apps/api/views/escalation_policy.py

```diff
@@ -49,31 +94,20 @@ class EscalationPolicyView(
         "move_to_position": [RBACPermission.Permissions.ESCALATION_CHAINS_WRITE],
     }
 
+    queryset = EscalationPolicy.objects.none()  # needed for drf-spectacular introspection
+
     model = EscalationPolicy
     serializer_class = EscalationPolicySerializer
     update_serializer_class = EscalationPolicyUpdateSerializer
     create_serializer_class = EscalationPolicyCreateSerializer
 
+    filter_backends = (filters.DjangoFilterBackend,)
+    filterset_class = EscalationPolicyFilter
+
     TEAM_LOOKUP = "escalation_chain__team"
 
     def get_queryset(self, ignore_filtering_by_available_teams=False):
-        escalation_chain_id = self.request.query_params.get("escalation_chain")
-        user_id = self.request.query_params.get("user")
-        slack_channel_id = self.request.query_params.get("slack_channel")
-        channel_filter_id = self.request.query_params.get("channel_filter")
```

Comment:
I think these are no longer used, can we remove them?

Reply:
They have been already removed

Reply:
https://github.com/grafana/oncall/pull/5037/files#diff-232bde1ff9834bf2d4fea690e3c5c94f77acd148ac0579b4081a8b2f43c37831R46-R56

I mean that frontend/mobile app don't seem to use these filters, can we remove them?

Reply:
got it, yeah, makes sense 👍